
Xvnc Free Edition 4.1.1 - built Feb 25 2015 23:02:21
Copyright (C) 2002-2005 RealVNC Ltd.
See http://www.realvnc.com for information on VNC.
Underlying X server release 40300000, The XFree86 Project, Inc


Fri Nov 29 11:27:41 2019
 vncext:      VNC extension running!
 vncext:      Listening for VNC connections on port 5901
 vncext:      created VNC server for screen 0
error opening security policy file /etc/X11/xserver/SecurityPolicy
Could not init font path element /usr/X11R6/lib/X11/fonts/Type1/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/Speedo/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/misc/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/75dpi/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/100dpi/, removing from list!
Could not init font path element /usr/share/fonts/X11/75dpi/, removing from list!
Could not init font path element /usr/share/fonts/X11/100dpi/, removing from list!
xrdb: No such file or directory
xrdb: can't open file '/home/is28/pavlov28/.Xresources'
/usr/bin/startxfce4: X server already running on display :1
gpg-agent[64239]: WARNING: "--write-env-file" is an obsolete option - it has no effect
gpg-agent: a gpg-agent is already running - not starting a new one

(xfce4-session:64232): xfce4-session-WARNING **: 11:27:44.894: gpg-agent returned no PID in the variables

(xfce4-session:64232): xfce4-session-WARNING **: 11:27:44.895: xfsm_manager_load_session: Something wrong with /home/is28/pavlov28/.cache/sessions/xfce4-session-virt-1:1, Does it exist? Permissions issue?

(xfwm4:64242): xfwm4-WARNING **: 11:27:44.917: The display does not support the XRender extension.

(xfwm4:64242): xfwm4-WARNING **: 11:27:44.917: The display does not support the XComposite extension.

(xfwm4:64242): xfwm4-WARNING **: 11:27:44.917: The display does not support the XDamage extension.

(xfwm4:64242): xfwm4-WARNING **: 11:27:44.917: The display does not support the XFixes extension.

(xfwm4:64242): xfwm4-WARNING **: 11:27:44.917: Compositing manager disabled.
(uint32 2,)

** (light-locker:64251): ERROR **: 11:27:44.930: Environment variable XDG_SESSION_PATH not set. Is LightDM running?

(polkit-gnome-authentication-agent-1:64253): GLib-CRITICAL **: 11:27:44.947: g_variant_new_string: assertion 'string != NULL' failed

(polkit-gnome-authentication-agent-1:64253): polkit-gnome-1-WARNING **: 11:27:44.948: Failed to register client: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.SessionManager was not provided by any .service files
xfwm4-Message: 11:27:44.962: Unsupported keyboard modifier '<Super>Tab'

(xfsettingsd:64257): xfsettingsd-CRITICAL **: 11:27:45.013: RANDR extension is too old, version 1.1. Display settings won't be applied.
Xlib:  extension "XInputExtension" missing on display ":1.0".

(xfsettingsd:64257): xfsettingsd-CRITICAL **: 11:27:45.014: XI is not present.

(xfsettingsd:64257): xfsettingsd-CRITICAL **: 11:27:45.015: Failed to initialize the Xkb extension.

(xfsettingsd:64257): xfsettingsd-CRITICAL **: 11:27:45.015: Failed to initialize the Accessibility extension.
xfwm4-Message: 11:27:45.053: Unsupported keyboard modifier '<Super>Tab'

(tracker-extract:64266): Tracker-CRITICAL **: 11:27:45.122: Could not request DBus name 'org.freedesktop.Tracker1.Miner.Extract': D-Bus service name:'org.freedesktop.Tracker1.Miner.Extract' is already taken, perhaps the application is already running?

(tracker-miner-apps:64309): Tracker-CRITICAL **: 11:27:45.137: Could not request DBus name 'org.freedesktop.Tracker1.Miner.Applications': D-Bus service name:'org.freedesktop.Tracker1.Miner.Applications' is already taken, perhaps the application is already running?

(nm-applet:64321): Gtk-WARNING **: 11:27:45.182: Can't set a parent on widget which has a parent

(tracker-miner-fs:64255): Tracker-CRITICAL **: 11:27:45.184: Could not request DBus name 'org.freedesktop.Tracker1.Miner.Files': D-Bus service name:'org.freedesktop.Tracker1.Miner.Files' is already taken, perhaps the application is already running?

(nm-applet:64321): Gdk-CRITICAL **: 11:27:45.319: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed

Fri Nov 29 11:28:01 2019
 Connections: accepted: 0.0.0.0::50233
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)

Fri Nov 29 11:28:03 2019
 VNCSConnST:  Server default pixel format depth 16 (16bpp) little-endian rgb565
 VNCSConnST:  Client pixel format depth 16 (16bpp) little-endian rgb565
/usr/lib/python3/dist-packages/blueman/plugins/applet/AppIndicator.py:8: PyGIWarning: AppIndicator3 was imported without specifying a version first. Use gi.require_version('AppIndicator3', '0.1') before import to ensure that the right version gets loaded.
  from gi.repository import AppIndicator3 as girAppIndicator
ERROR:dbus.proxies:Introspect error on org.bluez:/org/bluez: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.6319" (uid=1280 pid=64311 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)
ERROR:dbus.proxies:Introspect error on org.bluez:/: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.6319" (uid=1280 pid=64311 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)
ERROR:dbus.proxies:Introspect error on org.bluez:/: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.6319" (uid=1280 pid=64311 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)

Fri Nov 29 12:40:42 2019
 Connections: closed: 0.0.0.0::50233 (write: Connection reset by peer (104))
 SMsgWriter:  framebuffer updates 66235
 SMsgWriter:    copyRect rects 33, bytes 528
 SMsgWriter:    hextile rects 9925688, bytes -256304841
 SMsgWriter:    raw bytes equivalent 1931027992, compression ratio -7.534107

Fri Nov 29 12:40:44 2019
 Connections: accepted: 0.0.0.0::50341
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)
 VNCSConnST:  Server default pixel format depth 16 (16bpp) little-endian rgb565

Fri Nov 29 12:40:45 2019
 VNCSConnST:  Client pixel format depth 16 (16bpp) little-endian rgb565
 VNCSConnST:  Client pixel format depth 16 (16bpp) little-endian rgb565

Fri Nov 29 12:45:06 2019
 Connections: closed: 0.0.0.0::50341 (write: Connection reset by peer (104))
 SMsgWriter:  framebuffer updates 1757
 SMsgWriter:    hextile rects 1838007, bytes 108885559
 SMsgWriter:    raw bytes equivalent 616010676, compression ratio 5.657414

Fri Nov 29 12:45:10 2019
 Connections: accepted: 0.0.0.0::50347
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)
 VNCSConnST:  Server default pixel format depth 16 (16bpp) little-endian rgb565
 VNCSConnST:  Client pixel format depth 16 (16bpp) little-endian rgb565
 VNCSConnST:  Client pixel format depth 16 (16bpp) little-endian rgb565

Fri Nov 29 12:46:05 2019
 Connections: closed: 0.0.0.0::50347 (read: Connection reset by peer (104))
 SMsgWriter:  framebuffer updates 373
 SMsgWriter:    hextile rects 367765, bytes 24052792
 SMsgWriter:    raw bytes equivalent 123671564, compression ratio 5.141672

Tue Dec  3 11:31:40 2019
 Connections: accepted: 0.0.0.0::50511
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)

Tue Dec  3 11:31:42 2019
 SConnection: AuthFailureException: Authentication failure
 Connections: closed: 0.0.0.0::50511 (Authentication failure)
xfce4-session: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.
xfwm4: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.

(xfsettingsd:64257): libxfce4ui-WARNING **: 11:32:29.404: ICE I/O Error

(xfsettingsd:64257): libxfce4ui-WARNING **: 11:32:29.404: Disconnected from session manager.

(xfce4-panel:64246): libxfce4ui-WARNING **: 11:32:29.404: ICE I/O Error

(xfce4-panel:64246): libxfce4ui-WARNING **: 11:32:29.405: Disconnected from session manager.

(xfce4-terminal:10042): libxfce4ui-WARNING **: 11:32:29.404: ICE I/O Error

(xfce4-terminal:10042): libxfce4ui-WARNING **: 11:32:29.405: Disconnected from session manager.

(xfdesktop:64250): libxfce4ui-WARNING **: 11:32:29.405: ICE I/O Error

(xfdesktop:64250): libxfce4ui-WARNING **: 11:32:29.405: Disconnected from session manager.
Gdk-Message: 11:32:29.408: update-notifier: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.

xfdesktop: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.
Gdk-Message: 11:32:29.409: polkit-gnome-authentication-agent-1: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.

xfsettingsd: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.
Thunar: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.
xfce4-panel: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.
Gdk-Message: 11:32:29.416: evolution-alarm-notify: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.

Gdk-Message: 11:32:29.416: nm-applet: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.

wrapper-1.0: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.
wrapper-1.0: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.
Gdk-Message: 11:32:29.416: blueman-applet: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.

Gdk-Message: 11:32:29.415: xfce4-terminal: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.

medit: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.0.
/usr/share/system-config-printer/applet.py:45: PyGIWarning: Notify was imported without specifying a version first. Use gi.require_version('Notify', '0.7') before import to ensure that the right version gets loaded.
  from gi.repository import Notify
system-config-printer-applet: failed to start NewPrinterNotification service
system-config-printer-applet: failed to start PrinterDriversInstaller service: org.freedesktop.DBus.Error.AccessDenied: Connection ":1.6318" is not allowed to own the service "com.redhat.PrinterDriversInstaller" due to security policies in the configuration file
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 11 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 15 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 16 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 20 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 24 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 26 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 28 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 30 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 34 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 36 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 40 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 41 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 45 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 47 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 49 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 53 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 55 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 59 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 60 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 64 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 66 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 67 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 71 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 73 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 74 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 78 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 80 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 82 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 86 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 87 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 91 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 93 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 94 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 98 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 100 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 101 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 102 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 103 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 104 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 106 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 108 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 110 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 111 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 115 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 117 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 118 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 122 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 124 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 126 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 130 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 132 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 138 was not found when attempting to remove it
  GLib.source_remove (self.timer)
